R/utils.R

```diff
@@ -0,0 +1,32 @@
+#' Convert filter type to API filter type
+#'
+#' @description
+#' The API uses a slightly different naming convention for the different types of
+#' filters to what is used by analysts within data files. The function just converts
+#' from the file versions to the API versions.
+#'
+#' @inheritParams parse_tourl_filter_in
+#'
+#' @return String containing API friendly filter type descriptor
+#' @export
+#'
+#' @examples
+#' to_api_filter_type("filter_items")
+#' to_api_filter_type("geographic_levels")
+#' to_api_filter_type("locations")
+#' to_api_filter_type("filter_items")
+to_api_filter_type <- function(filter_type) {
+  if (!(filter_type %in% c("time_periods", "geographic_levels", "locations", "filter_items"))) {
```

Comment:
Could you be using the `validate_filter_type()` helper here instead? Looks like the same code?